packages/toolpad-app/src/server/localMode.ts

```diff
@@ -1165,8 +1150,6 @@ async function initToolpadFolder(root: string) {
     };
     await writeProjectFolder(root, projectFolder);
   }
-
-  await initGitignore(root);
```

Comment:
We can keep this file for now, it makes sure the content of the `.generated` folder is ignored.